Makefile.RAHasher

```diff
@@ -47,6 +50,10 @@ $(OUTDIR)/RAHasher$(EXE): $(OBJS)
 src/Git.cpp: etc/Git.cpp.template FORCE
 	cat $< | sed s/GITFULLHASH/`git rev-parse HEAD | tr -d "\n"`/g | sed s/GITMINIHASH/`git rev-parse HEAD | tr -d "\n" | cut -c 1-7`/g | sed s/GITRELEASE/`git describe --tags | sed s/\-.*//g | tr -d "\n"`/g > $@
 
+zip:
+	rm -f $(OUTDIR)/RALHasher-*.zip RAHasher-*.zip
```

Comment:
There's an errant `L` in your filename. You probably don't need the `$(OUTDIR)` item anyway.